### PR TITLE
[storage] [lint] Start enforcing lint errors

### PR DESCRIPTION
**Pull request by glimchb**
name, count, comments, ... see #195

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: storage/v1alpha1/backend_nvme_tcp.proto [hunk 1]
```diff
@@ -109,3 +83,27 @@ message NVMfRemoteControllerStatsResponse {
     string stats = 2;
 }
 
+enum NvmeTransportType {
```
Comment (glimchb) [inline storage/v1alpha1/backend_nvme_tcp.proto:86]: https://linter.aip.dev/126/unspecified

File: storage/v1alpha1/frontend_nvme_pcie.proto [hunk 2]
```diff
@@ -192,13 +188,13 @@ message NVMeNamespace {
 
     // Size/Capacity of the namespace in blocks, size in bytes will
     // be BlockSize x NumBlocks.
-    int64 num_blocks = 6;
+    int64 blocks_count = 6;
```
Comment (glimchb) [inline storage/v1alpha1/frontend_nvme_pcie.proto:191]: https://linter.aip.dev/141/count-suffix

File: storage/v1alpha1/backend_aio.proto [hunk 3]
```diff
@@ -30,11 +29,10 @@ message AioController {
     // handle will be returned with created object
     common.v1.ObjectKey handle = 1;
 
-    string name = 2;
-    int64 block_size = 3;
-    int64 num_blocks = 4;
```
Comment (glimchb) [inline storage/v1alpha1/backend_aio.proto:35]: https://linter.aip.dev/141/count-suffix

File: storage/v1alpha1/backend_null.proto [hunk 4]
```diff
@@ -25,10 +24,9 @@ service NullDebugService {
 
 message NullDebug {
     int64 id = 1;
-    string name = 2;
-    int64 block_size = 3;
-    int64 num_blocks = 4;
```
Comment (glimchb) [inline storage/v1alpha1/backend_null.proto:30]: https://linter.aip.dev/141/count-suffix

File: storage/v1alpha1/backend_nvme_tcp.proto [hunk 5]
```diff
@@ -33,31 +30,8 @@ message NVMfRemoteController {
     bool hdgst = 7;
     bool ddgst = 8;
     NvmeMultipath multipath = 9;
-    int64 num_io_queues = 10;
```
Comment (glimchb) [inline storage/v1alpha1/backend_nvme_tcp.proto:36]: https://linter.aip.dev/141/count-suffix

File: storage/v1alpha1/common.proto [hunk 6]
```diff
@@ -24,7 +24,7 @@ message NvmeControllerPciId {
     int32 device           = 2;
 
     // Physical function, always set to 0 in current model
-    int32 function         = 3;
```
Comment (glimchb) [inline storage/v1alpha1/common.proto:27]: https://linter.aip.dev/140/reserved-words

File: storage/v1alpha1/frontend_nvme_pcie.proto [hunk 7]
```diff
@@ -152,7 +149,7 @@ message NVMeController {
     // If not set, the xPU will provide a default.
     int32 max_nsq = 5;
 
-    // maximum number of host completion queues allowed. Optional.
```
Comment (glimchb) [inline storage/v1alpha1/frontend_nvme_pcie.proto:155]: https://linter.aip.dev/203/optional

File: storage/v1alpha1/frontend_nvme_pcie.proto [hunk 8]
```diff
@@ -192,13 +188,13 @@ message NVMeNamespace {
...
     // be BlockSize x NumBlocks.
-    int64 num_blocks = 6;
+    int64 blocks_count = 6;
 
     // Globally unique identifier for the namespace
     string nguid = 7;
 
     // 64bit Extended unique identifier for the namespace
-    // mandatory if guid is not specified, optional otherwise
+    // mandatory if guid is not specified
```
Comment (glimchb) [inline storage/v1alpha1/frontend_nvme_pcie.proto:197]: https://linter.aip.dev/203/optional

**Review by seroyer — APPROVED** [review]

**Pull request merged**